quill-core/src/main/scala/io/getquill/util/Messages.scala

```diff
@@ -4,6 +4,11 @@ import scala.reflect.macros.blackbox.{ Context => MacroContext }
 
 object Messages {
 
+  private val debugEnabled =
```

Comment:
It seems that this will disable the logs by default, is it intentional? I'd prefer to keep them enabled by default.